staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/controller.go

```diff
@@ -610,18 +611,20 @@ func CheckRequest(quotas []corev1.ResourceQuota, a admission.Attributes, evaluat
 		hardResources := quota.ResourceNames(resourceQuota.Status.Hard)
 		requestedUsage := quota.Mask(deltaUsage, hardResources)
 		newUsage := quota.Add(resourceQuota.Status.Used, requestedUsage)
-		maskedNewUsage := quota.Mask(newUsage, quota.ResourceNames(requestedUsage))
-
-		if allowed, exceeded := quota.LessThanOrEqual(maskedNewUsage, resourceQuota.Status.Hard); !allowed {
-			failedRequestedUsage := quota.Mask(requestedUsage, exceeded)
-			failedUsed := quota.Mask(resourceQuota.Status.Used, exceeded)
-			failedHard := quota.Mask(resourceQuota.Status.Hard, exceeded)
-			return nil, admission.NewForbidden(a,
-				fmt.Errorf("exceeded quota: %s, requested: %s, used: %s, limited: %s",
-					resourceQuota.Name,
-					prettyPrint(failedRequestedUsage),
-					prettyPrint(failedUsed),
-					prettyPrint(failedHard)))
+
+		if !skipCheck {
```

Comment:
So, this skips quota check on status update altogether for PVCs. Is that what we want? I thought we agreed that, we will only skip if object is _over_ quota and status needs to be written.

Reply:
If not exceeded, the quota will be updated. If exceeded and allowed to exceed, the quota will be updated. 

It is equivalent to the above code. It reduces unnecessary computation process.